### PR TITLE
deployment: update poetry

### PR DESCRIPTION
**Pull request by rerowep**
* Adds support for poetry > 1.0.10.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

**Review by Garfield-fr — APPROVED** [review]

File: scripts/test [hunk 1]
```diff
@@ -50,5 +50,5 @@ autoflake -c -r --remove-all-unused-imports --ignore-init-module-imports . &> /d
   exit 1
 }
 sphinx-build -qnNW docs docs/_build/html
-pytest
+pytest --color=yes
```
Comment (jma) [inline scripts/test:53]: Maybe, it would be better to put this in the `pytest.ini` file.

**Review by jma — APPROVED** [review]

**Comment by PascalRepond** [comment]
During static files collection in bootstrap, I get this errror : 

```
File "/home/rep/.cache/pypoetry/virtualenvs/sonar-Gg344htM-py3.9/lib/python3.9/site-packages/wand/api.py", line 175, in <module>
    raise ImportError('MagickWand shared library not found.\n'
ImportError: MagickWand shared library not found.
You probably had not installed ImageMagick library.
```

**Comment by PascalRepond** [comment]
After installing [wand](https://docs.wand-py.org/en/latest/guide/install.html), I got it working. If it is a prerequisite, it could be documented [install.rst](https://github.com/rerowep/sonar/blob/wep-poetry/INSTALL.rst)
Now, I get another error during setup: 
```
 Create organisations 
Usage: invenio fixtures organisations import [OPTIONS] FILE
Try 'invenio fixtures organisations import --help' for help.

Error: Invalid value for 'FILE': Could not open file: ./data/backups/organisations/data.json: No such file or directory
```

**Comment by PascalRepond** [comment]
Tested on Ubunutu 20.04 (WSL2)/Python 3.9.7

* [x] uses Poetry 1.1.13 (run `poetry self update` if needed)
* [x] ran `poetry run ./scripts/bootstrap`
* [x] ran `docker-compose up -d`
* [x] ran `poetry run ./scripts/setup --clear-data`

**Review by PascalRepond — APPROVED** [review]

**Pull request merged**